query/src/sql/planner/metadata.rs

```diff
@@ -84,15 +89,10 @@ pub struct Metadata {
 
 impl Metadata {
     pub fn create() -> Self {
-        Self::default()
-    }
-
-    fn next_table_index(&self) -> IndexType {
-        self.tables.len()
-    }
-
-    fn next_column_index(&self) -> IndexType {
-        self.columns.len()
+        Self {
```

Comment:
How about implementing `Default` for `Metadata`?

Reply:
Ditto, the `Default` trait doesn't seem reasonable to me.